Main.java

```diff
@@ -10,6 +8,19 @@
    */
   public static void main(String[] args) {
     // TODO code application logic here
-    
-  }
-}
+   Scanner input = new Scanner(System.in);
+  
+   //asks for number to divivde
+    System.out.println("Please enter in two integers, on seperate lines, to divide.");
+  
+  // Variable A is one number needed for division
+  int A = input.nextInt();
+
+  //variable B is the second number needed for division 
+  int B = input.nextInt();
+
+// declare and calculate the division
+System.out.println( A+ "/" + B +" is " + A / B + " with a remainder of " + A % B );
```

Comment:
While the math and everything works fine, try to avoid doing math in an output statement, Instead, create a variable that holds the math and then use that inside of the output statement. Also remember that variable names should always begin with a lowercase letter and be named after what they are storing or what they represent. In this case, a and b are ok names since they are just numbers but it might be better to call them numberA and numberB or something along those lines.